pkg/guacrest/server/retrieveDependencies_test.go

```diff
@@ -482,151 +480,41 @@ func Test_RetrieveDependencies(t *testing.T) {
 			gqlClient := SetupTest(t)
 			Ingest(ctx, t, gqlClient, tt.data)
 
-			restApi := server.NewDefaultServer(gqlClient)
+			encodedPurl := url.QueryEscape(tt.purl)
+			encodedArtifact := url.QueryEscape(tt.digest)
```

Comment:
See the other comments on escaping, but this shouldn't be needed here.

Reply:
Removed these escaping commands.